google/cloud/forseti/common/gcp_api/securitycenter.py

```diff
@@ -138,15 +139,27 @@ def create_finding(self, finding, source_id=None, finding_id=None):
                 '{}/findings/{}'.format(source_id, finding_id),
                 finding
             )
-            LOGGER.debug('Created finding response: %s',
+            LOGGER.debug('Successfully created finding response: %s',
                          response)
             return response
         except (errors.HttpError, HttpLib2Error) as e:
-            LOGGER.exception(
-                'Unable to create CSCC finding: Resource: %s', finding)
-            violation_data = (
-                finding.get('source_properties').get('violation_data'))
-            raise api_errors.ApiExecutionError(violation_data, e)
+            arguments = e.args
+            all_args = []
+            for argument in arguments:
```

Comment:
Why we be looping these arguments? Why not just go:
```
raw_error = e.args[1]
```

Reply:
Done